lib/version_utils.pm

```diff
@@ -744,6 +745,21 @@ sub check_os_release {
     return ($os_like_name =~ /$distri_name/i);
 }
 
+=head2 verify_os_version
+
+Returns 1 (true) if os release version matches the one passed as arguement.
+If no arguements are given, the function will compare the os release version
+in /etc/os-release file with "VERSION" var.
+
+=cut
+
+sub verify_os_version {
+    my ($version, $os_release_file) = @_;
+    $version //= get_var("VERSION");
+    $os_release_file //= '/etc/os-release';
```

Comment:
You're not using this variable.

Reply:
Right, too sloppy copying pasting. Thanks